Tests/OAuth/ResourceOwner/AzureResourceOwner.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the HWIOAuthBundle package.
+ *
+ * (c) Hardware.Info <opensource@hardware.info>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace HWI\Bundle\OAuthBundle\Tests\OAuth\ResourceOwner;
+
+use HWI\Bundle\OAuthBundle\OAuth\ResourceOwner\AzureResourceOwner;
+
+class AzureResourceOwnerTest extends GenericOAuth2ResourceOwnerTest
+{
+    protected $csrf = true;
+
+    protected $userResponse = <<<json
+{
+    "id":  "1",
+    "foo": "bar"
```

Comment:
@Soullivaneuh 
In that case it could be something like:

{
    "aud": "user-identifier",
    "upn": "user-identifier",
    "sub": "fxoF978Q-epzh-0_9Yq5zE2sN_KXFylp0PEsSWqcjvU",
    "given_name": "Dummy",
    "family_name": "Tester",
    "name": "Dummy Tester",
    "amr": ["pwd"],
    "unique_name": "dummy@tester.org",
    "pwd_exp": "7700986",
    "pwd_url": "https://portal.microsoftonline.com/ChangePassword.aspx"
}

What's the quickest way to get it in here?